### PR TITLE
Add solution hint to error message

### PR DESCRIPTION
**Pull request by alexgparity**

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable-int
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2244661

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2244656

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2244655

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-check-benches
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2244651

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-check-each-crate
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2244665

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-check-benches
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2244650

File: frame/support/procedural/src/pallet/expand/storage.rs [hunk 1]
```diff
@@ -535,7 +535,7 @@ pub fn expand_storages(def: &mut Def) -> proc_macro2::TokenStream {
 							<T as #frame_system::Config>::PalletInfo
 							as #frame_support::traits::PalletInfo
 						>::name::<Pallet<#type_use_gen>>()
-							.expect("Every active pallet has a name in the runtime; qed")
+							.expect("Every active pallet has a name in the runtime; qed. Have you added the pallet into construct_runtime! ?")
```
Comment (bkchr) [inline frame/support/procedural/src/pallet/expand/storage.rs:538]:
```suggestion
							.expect("No name found for the pallet! This usually means that the pallet wasn't added to `construct_runtime!`.")
```

File: frame/support/procedural/src/pallet/expand/storage.rs [hunk 2]
```diff
@@ -569,7 +569,7 @@ pub fn expand_storages(def: &mut Def) -> proc_macro2::TokenStream {
 						<T as #frame_system::Config>::PalletInfo
 						as #frame_support::traits::PalletInfo
 					>::name::<Pallet<#type_use_gen>>()
-						.expect("Every active pallet has a name in the runtime; qed")
+						.expect("Every active pallet has a name in the runtime; qed. Have you added the pallet into construct_runtime! ?")
```
Comment (bkchr) [inline frame/support/procedural/src/pallet/expand/storage.rs:572]:
```suggestion
						.expect("No name found for the pallet! This usually means that the pallet wasn't added to `construct_runtime!`.")
```

File: frame/support/procedural/src/pallet/expand/storage.rs [hunk 3]
```diff
@@ -648,7 +648,7 @@ pub fn expand_storages(def: &mut Def) -> proc_macro2::TokenStream {
 						<T as #frame_system::Config>::PalletInfo as
 						#frame_support::traits::PalletInfo
 					>::name::<#pallet_ident<#type_use_gen>>()
-						.expect("Every active pallet has a name in the runtime; qed"),
+						.expect("Every active pallet has a name in the runtime; qed. Have you added the pallet into construct_runtime! ?"),
```
Comment (bkchr) [inline frame/support/procedural/src/pallet/expand/storage.rs:651]:
```suggestion
						.expect("No name found for the pallet! This usually means that the pallet wasn't added to `construct_runtime!`."),
```

**Review by bkchr — APPROVED** [review]

**Review by tdimitrov — APPROVED** [review]

**Comment by alexgparity** [comment]
bot merge

**Pull request merged**